department-of-veterans-affairs/caseflow
# Do not load attorneys if props don't tell us to

Resolves unnecessarily loading attorneys.

### Description
`!(true && true)` -> `!true || !true`. Who knew?

More importantly, we do not need a full list of attorneys when a judge is in their "review" queue. We do not use this information at all on the review page.

### Acceptance Criteria
- [x] Attorneys are not loaded in a judge's review queue
- [x] Attorneys are loaded in a judge's "assign" queue
- [x] Attorneys are loaded when a judge visits case details

### Testing Plan
1. Log in as BVAAAAAABSHIRE
1. Go to your review queue http://localhost:3000/queue
1. Ensure there is NO call to /users
1. Click into a case
1. Ensure there IS a call to /users
1. Ensure your attorneys appear in the assign dropdown
1. Ensure all attorneys appear in the "Other" assign dropdown
1. Click back into queue
1. Ensure there is NO call to /users
1. Switch views to your assign queue
1. Ensure there IS a call to /users
1. Ensure your attorneys appear in the assign dropdown
1. Ensure all attorneys appear in the "Other" assign dropdown
